ui/components-react/src/components-react/filter-builder/FilterBuilderRule.scss

```diff
@@ -45,10 +26,22 @@
     width: 15rem;
     display: flex;
     flex-direction: column;
+
+    >span {
```

Comment:
yes because the `span` has the `components-editor-container` class and then the `input` nested beneath that has the `components-text-editor` class. instead of >span here I could use `.components-editor-container`

Reply:
I am still not sure why `>span` or `.components-editor-container` is necessary. `.fb-property-value .components-text-editor` should have higher  specificity than `.components-text-editor`. So it should be enough to have something like:
```
.fb-property-value {
  .components-text-editor {
    height: unset;
  }
}
```

Reply:
ah you're right, I was thinking I needed to target the exact hierarchy so 
`.fb-property-value` > `.components-editor-container` > `.components-text-editor`

but removing it works so I will push that now